semgrep_output_v1.atd

```diff
@@ -1803,6 +1813,7 @@ type function_call
   | CallApplyFixes of apply_fixes_params
   | CallSarifFormat of sarif_format_params
   | CallFormatter of (output_format * cli_output)
+  | CallValidate of validation_params
```

Comment:
I don't think you need to define an intermediate type when the type is simple no @nmote ?
Here it could be just CallValidate of fpath (* temporary filename containing all the rules *)